scipy/stats/stats.py

```diff
@@ -3403,7 +3400,7 @@ def ttest_ind(a, b, axis=0, equal_var=True):
     then we cannot reject the null hypothesis of identical average scores.
     If the p-value is smaller than the threshold, e.g. 1%, 5% or 10%,
     then we reject the null hypothesis of equal averages.
-
+        
```

Comment:
this is empty whitespae added. you may want to configure your editor to strip all whitespace when saving a file